dashboard/app/models/pd/enrollment.rb

```diff
@@ -103,6 +103,18 @@ def completed_survey?
     PEGASUS_DB[:forms].where(kind: 'PdWorkshopSurvey', source_id: id).any?
   end
 
+  def local_summer?
+    workshop.course == Pd::Workshop::COURSE_CSP && workshop.subject == Pd::Workshop::SUBJECT_CSP_SUMMER_WORKSHOP
+  end
```

Comment:
I seriously debated that; it's a property of the workshop, but only the enrollment cares about that property. I guess since the question of who cares is much more likely to change than the question of what influences this, I should go ahead and move it.